mypy/errors.py

```diff
@@ -825,7 +830,10 @@ def load_baseline(self, file: Path) -> bool:
 
         if not file.exists():
             return False
+        from copy import deepcopy
+
```

Comment:
why's this imported here and not at the top of the file

Reply:
umm, it's not